core/templates/dev/head/pages/collection_editor/editor_tab/collection_node_creator_directive.html

```diff
@@ -5,6 +5,6 @@
 </form>
 or
 <form ng-submit="addExploration()">
-  <input ng-model="newExplorationId" type="text" placeholder="New exploration ID">
-  <button type="submit" ng-disabled="!newExplorationId">Add Existing Exploration</button>
+  <input class="form-inline" ng-model="newExplorationId" type="text" placeholder="New exploration ID">
```

Comment:
You know, that is a good point @zvekovius (I assume you're referring to what is visibile by the user). I'd lean towards "Existing exploration ID" to match the button text. What do you think?

I think keeping the internal variable as newExplorationId is fine.